run.bat

```diff
@@ -0,0 +1,28 @@
+echo >/dev/null # >nul & GOTO WINDOWS & rem ^
+
+export REGISTRY_NAME=
```

Comment:
Nice. I am missing instructions to populate the optional params.

Reply:
The export I'm doing here?
Those are just so that warnings won't show when running compose.
The REGISTRY_NAME, IMAGE_PREFIX, TAG are for the CI process and should be changed during local build.